src/test/kotlin/no/nav/helsearbeidsgiver/testcontainer/Containers.kt

```diff
@@ -0,0 +1,61 @@
+package no.nav.helsearbeidsgiver.testcontainer
+
+import org.junit.jupiter.api.extension.AfterAllCallback
+import org.junit.jupiter.api.extension.BeforeAllCallback
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.ExtensionContext
+import org.testcontainers.containers.PostgreSQLContainer
+import org.testcontainers.containers.wait.strategy.Wait
+import org.testcontainers.kafka.KafkaContainer
+import org.testcontainers.utility.DockerImageName
+
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.RUNTIME)
+@ExtendWith(PostgresTestExtension::class)
+annotation class WithPostgresContainer
+
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.RUNTIME)
+@ExtendWith(KafkaTestExtension::class)
+annotation class WithKafkaContainer
+
+class PostgresTestExtension :
+    BeforeAllCallback,
+    AfterAllCallback {
+    companion object {
+        private val postgresContainer =
+            PostgreSQLContainer("postgres:15-alpine")
+                .withDatabaseName("testdb")
+                .withUsername("testuser")
+                .withPassword("testpass")
+                .waitingFor(Wait.forListeningPort())
+    }
+
+    override fun beforeAll(context: ExtensionContext) {
+        postgresContainer.start()
+        System.setProperty("database.url", postgresContainer.jdbcUrl)
+        System.setProperty("database.username", postgresContainer.username)
+        System.setProperty("database.password", postgresContainer.password)
+    }
+
+    override fun afterAll(context: ExtensionContext) {
+        postgresContainer.stop()
+    }
+}
+
+class KafkaTestExtension :
+    BeforeAllCallback,
+    AfterAllCallback {
+    companion object {
+        private val kafkaContainer = KafkaContainer(DockerImageName.parse("apache/kafka:latest"))
```

Comment:
stemmer for å bruke confluent sin testcontainer for kafka:
https://github.com/navikt/helsearbeidsgiver-inntektsmelding/blob/76e6a238b948fa15266df0f0b532cc0a43cc4762/apps/integrasjonstest/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/integrasjonstest/utils/ContainerTest.kt#L19